test/fops/file.c

```diff
@@ -27,6 +27,7 @@
  */
 
 #include "thread.h"
+#include "../utility/util.h"
```

Comment:
Best not to use relative paths here. You should update the ```AM_CPPFLAGS``` line in ```test/fops/Makefile.am``` to include ```-I$(top_srcdir)/test/utility``` and remove the path here.